sql/src/main/java/io/crate/action/sql/SQLOperations.java

```diff
@@ -205,6 +212,7 @@ public void parse(String statementName, String query, List<DataType> paramTypes)
                 this.statement = statement;
                 this.query = query;
                 this.paramTypes = paramTypes;
+                statsTables.jobStarted(jobId, query);
```

Comment:
this should imho only be done in the if and else if block, otherwise bulk operations will generate mutliple job entries